### PR TITLE
ADEN-6719 Clean up MBLB leftovers

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 1]
```diff
@@ -48,7 +48,6 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverBabRecoveryCountries';
 		$vars[] = 'wgAdDriverBeachfrontBidderCountries';
 		$vars[] = 'wgAdDriverBottomLeaderBoardMegaCountries';
-		$vars[] = 'wgAdDriverBottomLeaderBoardOnMobileCountries';
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: We shouldn't remove this instant global in the same PR. IG cache will be invalidated earlier and we'll disable BLB on mobile for 24h. Am I right?

File: extensions/wikia/AdEngine/js/lookup/services.js [hunk 2]
```diff
@@ -32,24 +31,13 @@ define('ext.wikia.adEngine.lookup.services', [
 			}
 		},
 		bidMarker = ['x', 'x', 'x', 'x', 'x'],
-		realSlotPrices = {},
-		slotAliases = {
-			mercury: {
-				BOTTOM_LEADERBOARD: 'MOBILE_PREFOOTER',
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/services.js:38]: What about all prebid adapters? We should change MOBILE_PREFOOTER to BOTTOM_LEADERBOARD there. Is it safe to do now?

File: extensions/wikia/AdEngine/js/slot/slotTargeting.js [hunk 3]
```diff
@@ -33,8 +33,7 @@ define('ext.wikia.adEngine.slot.slotTargeting', [
 
 			MOBILE_TOP_LEADERBOARD: 'l',
 			MOBILE_IN_CONTENT: 'i',
-			MOBILE_PREFOOTER: 'p',
-			MOBILE_BOTTOM_LEADERBOARD: 'b'
+			MOBILE_PREFOOTER: 'p'
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/slot/slotTargeting.js]: Should we remove MOBILE_PREFOOTER?

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js [hunk 4]
```diff
@@ -41,7 +41,7 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 			'LB': ['TOP_LEADERBOARD', 'MOBILE_TOP_LEADERBOARD'],
 			'MR': ['TOP_RIGHT_BOXAD'],
 			'PF': [
-				'MOBILE_PREFOOTER', 'BOTTOM_LEADERBOARD', 'MOBILE_BOTTOM_LEADERBOARD'
+				'MOBILE_PREFOOTER', 'BOTTOM_LEADERBOARD'
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: Should we remove MOBILE_PREFOOTER?

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: Hmm, that is true. I'm gonna split this into two PR.

File: extensions/wikia/AdEngine/js/lookup/services.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/services.js:38]: I gonna change `MOBILE_PREFOOTER` to `BOTTOM_LEADERBOARD` in all Prebid.js adapters and ask Nef if that is gonna be fine.

File: extensions/wikia/AdEngine/js/provider/directGptMobile.js [hunk 5]
```diff
@@ -15,10 +15,7 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
 			INVISIBLE_HIGH_IMPACT_2:    {loc: 'hivi'},
 			MOBILE_TOP_LEADERBOARD:     {size: '300x50,320x50,320x100,320x480,2x2', loc: 'top'},
-			// TODO remove in ADEN-6719
-			MOBILE_BOTTOM_LEADERBOARD:  {size: '320x50,300x250,300x50,2x2', loc: 'footer', pos: ['MOBILE_BOTTOM_LEADERBOARD', 'BOTTOM_LEADERBOARD', 'MOBILE_PREFOOTER']},
-			// TODO remove pos attribute in ADEN-6719
-			BOTTOM_LEADERBOARD:         {size: '320x50,300x250,300x50,2x2', loc: 'footer', pos: ['BOTTOM_LEADERBOARD', 'MOBILE_BOTTOM_LEADERBOARD', 'MOBILE_PREFOOTER']},
+			BOTTOM_LEADERBOARD:         {size: '320x50,300x250,300x50,2x2', loc: 'footer'},
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/provider/directGptMobile.js]: I believe we need to send `pos:mobile_prefooter` unless we want to reetarget all prebid lines in DFP

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/provider/directGptMobile.js]: I think you right, added.

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js (hunk 4 above)
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js:44]: What are our plans towards changing `'PF'` group in future? For example to `'BL'` ;)

**Comment by nandy-andy** [comment]
Are we supposed to rebuild `bridge.js`? It still has occurrences of `MOBILE_BOTTOM_LEADERBOARD`.

**Comment by dmnsobczak** [comment]
You right. Bridge rebuilded.

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js (hunk 4 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js:44]: Right now we don't have to and we didn't plan to to this, but sure think we can groom it on next occasion if we want to or should to do this.

**Comment by fraszczakszymon** [comment]
What about https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/lookup/a9.js#L22

**Comment by dmnsobczak** [comment]
Changed too, thanks!

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**